### ISSUE TITLE
The categories show page is no longer relevant

### ISSUE THREAD

**Issue by kenmazaika** [FirehoseCommunity/Lets-Pair-Program#78]
Since we're listing out the information on the `categories#index` the `categories#show` can be removed.  Blocked by #75, don't do this prior to implementing that feature.

**Comment by cacauzen** [comment]
And also this one, to make things easier.

**Comment by amarkpark** [comment]
category_path (the categories#show page) is currently being called in the sections_controler create action, and it's being called by the "Add Section" button on the categories#index view... I think design options in the meantime may have negated this issue

**Comment by amarkpark** [comment]
Yeah so I finally figured out this morning the point of this post.  D'Oh!

**Comment by amarkpark** [comment]
Might be able to use following link to complete this task to my satisfaction:
https://api.jquery.com/target-selector/
